sources/packages/api/src/services/application/application.service.ts

```diff
@@ -572,7 +572,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
       );
     } else {
       applicationQuery.orderBy(
-        `CASE application.applicationStatus
+        `CASE application.application_status
```

Comment:
Please add a TODO for further investigation on why `application.applicationStatus` was not translated to the DB column name. This will be taken care is the next PR, right?